src/SFML/Window/CMakeLists.txt

```diff
@@ -2,6 +2,32 @@
 set(INCROOT ${PROJECT_SOURCE_DIR}/include/SFML/Window)
 set(SRCROOT ${PROJECT_SOURCE_DIR}/src/SFML/Window)
 
+set(WAYLAND_PROTOCOL_SRC)
+
+if(SFML_WAYLAND)
+	include_directories("${CMAKE_BINARY_DIR}/src")
```

Comment:
Since you want to access `src` directory, this should be `${CMAKE_SOURCE_DIR}`. Also, there are other places where it's wrong as well.

`CMAKE_BINARY_DIR` is where the binaries are. The user decides where they land.

Reply:
IIRC i wasn't including the source directory here. I'm producing wayland protocol headers which end up in `${CMAKE_BINARY_DIR}/src` not `$(CMAKE_SOURCE_DIR)`